### ISSUE TITLE
Blind user unable to turn screen reader on (no keyboard shortcut, not tabbable)

### ISSUE THREAD

**Issue by jacobgkau** [pop-os/cosmic-initial-setup#104]
https://github.com/pop-os/cosmic-initial-setup/issues/46 disabled the screen reader by default since most users don't need it and are annoyed by the voice starting by itself.

Our UX team intended for a `Super+Alt+S` shortcut to be able to turn the screen reader on, but that hasn't been implemented yet. The on/off switch for the screen reader is also not selectable with the keyboard using the tab key. Because of this, our support team had a customer contact where they were unable to help the blind customer enable the screen reader, since the customer doesn't have the necessary level of vision to be able to see the on/off switch to click it on.

We should implement the keyboard shortcut at least in COSMIC Initial Setup if not globally (it seems like globally would be ideal), or at least make the control tabbable to it can be found with trial and error/at the direction of remote support easier than having to blindly click.

**Comment by jacobgkau** [comment]
As a potential workaround, starting the launcher with Super and then typing `t:orca` (or `t: orca`) followed by Enter does launch a Terminal window with the screen reader, enabling it. (`:orca` to try and launch it headlessly does not enable it.)
